zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Identity.java

```diff
@@ -34,6 +33,8 @@
     private boolean datasetImportEnabled = false;
 
     static final String AUTHORIZATION_HEADER = "Authorization";
+    static final String BEARER_PREFIX_UC = "Bearer";
+    static final String BEARER_PREFIX_LC = "bearer";
```

Comment:
It doesn’t look like the lowercase version is being used anywhere (from what I can see on my phone anyway). Probably worth removing it for clarity. Could potentially remove the UC from the other contestant then.